network/peers/subnets.go

```diff
@@ -57,6 +57,8 @@ diffLoop:
 					if len(peers) == 1 {
 						si.subnets[subnet] = make([]peer.ID, 0)
 					} else {
+						// #nosec
+						// False positive by the linter, it says potential out of bounds, but it can't be because we're inside a for on `peers`.
```

Comment:
```suggestion
						// False positive by the linter, it says potential out of bounds,
						// but it can't be because we're inside a for on `peers`.
```